app/views/content/train-to-be-a-teacher/what-is-a-pgce.md

```diff
@@ -10,6 +11,9 @@ related_content:
 date: "2021-11-01"
 image: "static/content/hero-images/0015.jpg"
 backlink: "../../"
+navigation: 20.55
+navigation_title: Postgraduate certificate in education (PGCE)
+navigation_description: A PGCE can help develop your teaching skills and knowledge. You will also need English qualified teacher status (QTS) to teach in maintained schools in England.
```

Comment:
I don't think `also` is quite right here @philspray as it implies you **need** both, which is a bit confusing when read alongside the QTS nav description?

Would maybe suggest something like:

* Having a PGCE as well as qualified teacher status (QTS) can help you develop your teaching skills and knowledge.

However, I think this is one to check with policy